plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -29,6 +29,70 @@ FileSystemAdaptor.prototype.name = "filesystem";
 
 FileSystemAdaptor.prototype.supportsLazyLoading = false;
 
+/*
+Show a filesystem errorlog alert
+*/
+FileSystemAdaptor.prototype.displayError = function(msg,err) {
```

Comment:
There's a lot going on this method, does it need to be private to the file system adaptor? Also we already have a logging feature (see the tiddlyweb adaptor), can we not use that here?